conf/modules.config

```diff
@@ -98,7 +98,7 @@ if(params.aligner == "cellranger") {
                 path: "${params.outdir}/${params.aligner}/count",
                 mode: params.publish_dir_mode
             ]
-            ext.args = {"--chemistry ${meta.chemistry} " + (meta.expected_cells ? "--expect-cells ${meta.expected_cells}" : '')}
+            ext.args = {"--chemistry ${meta.chemistry} --create-bam false " + (meta.expected_cells ? "--expect-cells ${meta.expected_cells}" : '')}
```

Comment:
Why? E.g. RNA velocity analysis relies on this, I'd expect a couple of users complain about this being default all of a sudden.

Reply:
New version of cellranger requires this parameter to be used.

Did not know it was used by other workflows. Should I default to true or make a parameter?

Reply:
I see. A parameter would be nice, but I don't mind setting to true and following up on this separately.

Reply:
Now defaulting to true. Anyways, users can still overwrite if needed as it is a `ext.args`. So, we can consider having a param later upon request.